src/util/analytics-client.ts

```diff
@@ -1,7 +1,7 @@
 import { getLogger 	} from "config/logger";
 import { omit } from "lodash";
-import { optionalRequire } from "optional-require";
 import { isNodeProd } from "utils/is-node-env";
+import { optionalRequire } from "optional-require/dist";
```

Comment:
`optional-require` is already exporting everything inside of `dist/index.d`, so not sure if we need to add another `/dist`.